mews-operations/reservations.md

```diff
@@ -385,6 +385,32 @@ The third `reservation` definition shows the partial cancellation - cancelling t
 
 > **From/To:** This represents the reservation arrival, `from` is arrival date, `to` is departure date. So reservation for 2 nights (e.g. 2021-12-24/25 and 2021-12-25/26 is represented as `"from": "2021-12-24", "to": "2021-12-26"`).
 
+##### Delivery specific fields
+
```

Comment:
I must admit I am struggling with these sections. I think there are three reasons:
* The objects passed on both sides have their own unique definitions, there is no implication that they might be the same, so if they are different then nothing special is needed except perhaps a helpful note to put in the operation description.
* The docs is getting messy, it would be better to try and follow a simpler model where the API Reference section contains purely factual descriptions of the operations and data fields, and any discussion of how they are used is put into a separate section of guidance on using the API, outside of the API Reference. I also think this model will be a lot easier to maintain going forward.
* There is potential confusion in the CHM API because of having the two sides, especially as operations on both sides have the same name, so I think we have to follow a strict naming convention. My recommendation is, for example, `CHM: Process group` vs `Mews: Process group`

Reply:
yeah, the solution for it would be to "duplicate" the definition on each side. Right now the CHM side links the explanation on Mews side (same as in code ...).